docs/reference/api/drivers/Watchdog.md

```diff
@@ -1,11 +1,40 @@
 ## Watchdog
 
-[Add description here.]
+Use the Watchdog interface to set up a hardware watchdog that resets the system after a set period of time if you do not periodically refresh it before it expires.
+
+<span class="notes">**Note:** The maximum amount of time that you can set as the Watchdog timeout varies depending on the target hardware. You can check the maximum value by calling `Watchdog::max_timeout()`.</span>
 
 ### Watchdog class reference
 
-[Add class reference here. Ask your editor for help if you've never done this before.]
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/v5.8/mbed-os-api-doxy/classmbed_1_1_watchdog.html)
```

Comment:
This link doesn't work. Please fix it.

Reply:
Can you please help me find the correct link here: https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/annotated.html

Reply:
This? https://os-doc-builder.test.mbed.com/docs/development/feature-hal-spec-watchdog-doxy/watchdog__api_8h_source.html

Reply:
These are not contributing (porting) pages, so they should not go to feature branches.